src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/LinkWidget.tsx

```diff
@@ -62,7 +258,7 @@ export class DeletableLinkFactory extends AbstractLinkFactory {
   }
 
   generateReactWidget(diagramEngine, link) {
-    return <DeletableLinkWidget link={link} diagramEngine={diagramEngine} />
+    return <DeletableLinkWidget link={link} color="#bec5c9" width={2} diagramEngine={diagramEngine} />
```

Comment:
is there a way we apply a class instead of specifying color and witdh straight up "?